hbase-server/src/main/java/org/apache/hadoop/hbase/master/procedure/EnableTableProcedure.java

```diff
@@ -111,25 +104,16 @@ protected Flow executeFromState(final MasterProcedureEnv env, final EnableTableS
           // How many replicas do we currently have? Check regions returned from
           // in-memory state.
           int currentMaxReplica = getMaxReplicaId(regionsOfTable);
-
-          // Read the META table to know the number of replicas the table currently has.
-          // If there was a table modification on region replica count then need to
-          // adjust replica counts here.
-          int replicasFound = TableName.isMetaTableName(this.tableName)?
```

Comment:
We didn't actually use this replicasFound that we read from meta?

Reply:
It will be log in the line below but I do not see the necesity. It does not effect the actual logic, if there are inconsistency, we should report it in the report from CatalogJanitor and fix it with HBCK2? Just a simple log here will just make users confusing.